lib/video_info/providers/youtubeplaylist_scraper.rb

```diff
@@ -0,0 +1,31 @@
+require 'nokogiri'
+require 'open-uri'
+require 'open_uri_redirections'
+
+class VideoInfo
+  module Providers
+    module YoutubePlaylistScraper
+      def description
+	data.css('meta').find { |m| m.values[0] == 'description' }.values[1]
+      end
+      
+      def title
+	data.css('meta').find { |m| m.values[0] == 'title' }.values[1]
+      end
+      
+      def videos
+	raise(NotImplementedError, 'To access videos, you must provide an API key to VideoInfo.provider_api_keys')
```

Comment:
Line is too long. [107/80]<br>Use 2 (not -5) spaces for indentation.<br>Tab detected.